### DISCUSSION TITLE
Actual templating

### DISCUSSION THREAD

**Discussion by joshwithers** [docusealco/docuseal#25] [Ideas]
My clients need to sign Australian federal government paperwork with me, the PDF changes but the places they sign doesn't. It would be great to set up a template for what Docuseal currently calls templates, so I can upload their personal PDF and the signature boxes etc carry over.

**Comment by AlexBTurchyn** [comment]
@joshwithers Thanks for your suggestion.

I believe there's an alternative that might serve your needs. In the latest release (1.0.6), it's possible to replace a document in the template without losing any of the fields you've already set. 

So, what you could do is clone the template and replace the document in the clone. This way, all the previously marked fields would be preserved.

Hope you can give it a try and please let me know how it works for you.

![2023-07-28_00-18](https://github.com/docusealco/docuseal/assets/1176367/60f2afb5-7f8e-49bf-9c87-e20a115d5679)

**Reply by joshwithers** [reply]
Ahh that looks cool, I'll wait for that update to come to the Railway version

**Reply by AlexBTurchyn** [reply]
@joshwithers Railway version is not updated automatically, so your instance must be updated manually. I prepared [instructions](https://github.com/docusealco/docuseal/wiki/How-to-Update-DocuSeal-App-Installed-Using-the-%22Deploy-on-Railway%22-Button) on how to do it. Let me know if something is wrong.

**Comment by joshwithers** [comment]
thank you!

—
Josh
Insert a cool email signature here
withers.co
